layouts/_default/summary.html

```diff
@@ -8,9 +8,16 @@
 	{{- end }}
 	<div class="list__content clearfix">
 		<header class="list__header">
-			<h3 class="list__title post__title ">
-				<a href="{{ .RelPermalink }}" rel="bookmark">{{ .Title }}</a>
-			</h3>
+				<h3 class="list__title post__title ">
+					<a href="{{ .RelPermalink }}" rel="bookmark">
+					{{ .Title }}
+					</a>
+				</h3>
+				{{- if .Params.lead }}
```

Comment:
@pkel,

Use {{ with }}

Reply:
I take no offense if you edit further PRs directly.